traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/CategoryFacet.java

```diff
@@ -1,21 +1,20 @@
 package com.guestlogix.travelercorekit.models;
 
-import com.guestlogix.travelercorekit.utilities.ObjectMappingFactory;
-
 import com.guestlogix.travelercorekit.utilities.JSONObjectGLX;
+import com.guestlogix.travelercorekit.utilities.ObjectMappingFactory;
 
 import java.io.Serializable;
 
 public class CategoryFacet implements Serializable {
-    private BookingItemCategory category;
+    private String category;
```

Comment:
backend is sending category as String here :/ ? why didt they change here ? its so messy

Reply:
i asked them they havent changed the model here. put a todo here. we should fix this when they fixed it on their side.

Reply:
added TODO